src/custom/hooks/useFetchProfile.ts

```diff
@@ -6,19 +6,26 @@ import { ProfileData } from 'api/gnosisProtocol/api'
 export default function useFetchProfile() {
   const { account, chainId } = useActiveWeb3React()
   const [profileData, setProfileData] = useState<ProfileData | null>(null)
+  const [error, setError] = useState('')
```

Comment:
Rebase this branch on top of `ramirotw/issue-1632-profile-loader` so you can use this state https://github.com/gnosis/cowswap/pull/1753/files#diff-8d11b767719917b167894fe5731a9ba0282930692307a5c17c3f672262001814R8

Reply:
:+1: Done!